root/account/edit.tt

```diff
@@ -65,8 +65,8 @@
             </span>
             [% extra_field = form.field('languages').field(form.field('languages').add_extra(1) - 1) %]
             <span id="add-language-template" style="display: none">
-              [% r.select(extra_field.field('language_id'), id => 'fluency-template', class => 'language_id',) %]
-              [% r.select(extra_field.field('fluency'), class => 'fluency') %]
+              [% r.select(extra_field.field('language_id'), id => 'id-language-id-template', class => 'language_id', name => 'language-id-template') %]
+              [% r.select(extra_field.field('fluency'), id => 'id-fluency-template', class => 'fluency', name => 'fluency-template') %]
```

Comment:
Something I'm wondering is, are these id and name attributes actually needed? It seems they aren't used anywhere and are just overridden in the script below, so we can just remove them.